New option: --per-file-ignores (#3675)
codespell-project/codespell: README.rst

       def wrod(wrods) # codespell:ignore
           pass

Per-file ignores
----------------

> To give a finer control, is possible to specified a additional set of words to ignore into a specific file only.

Reviewer: ```suggestion
To give a finer control, it is possible to specify an additional set of words to ignore in a specific file.
```
